test/resources/math.js

```diff
@@ -148,12 +148,13 @@ function orderDisregardedUtility(order, executedBuyAmount, prices) {
  * @return {BN} The solution's objective value
  */
 function solutionObjectiveValue(orders, solution) {
-  return solutionObjectiveValueComputation(orders, solution).result
+  return solutionObjectiveValueComputation(orders, solution, true).result
```

Comment:
Do we want this to be kept here? or remove the true?
```suggestion
  return solutionObjectiveValueComputation(orders, solution).result
```

Reply:
The true was on purpose, the `solutionObjectiveValue` function wants to be strict. I made it explicit so that if the default parameter value changes, `solutionObjectiveValue` will keep the same behaviour.